core/pom.xml

```diff
@@ -127,6 +161,10 @@
       <version>5.3.0.RELEASE</version>
     </dependency>
 
+    <dependency>
+        <groupId>io.github.lognet</groupId>
```

Comment:
I am using https://github.com/feast-dev/feast/pull/793/commits/639c6079005e095837001d2394b65e1d1eaa7fdc#diff-357e4854869b2e21c38b1b437f11095aR116 for the spring boot gRPC starter, because Lognet has a security vulnerability. 

Is there a reason  you added it back?

Reply:
just the merge.  I'll try to remove.